services/api-server/src/simcore_service_api_server/models/raw_data.py

```diff
@@ -0,0 +1,5 @@
+from typing import Any, Dict, List, Union
+
+JsonDict = Dict[str, Any]
+JsonList = List[JsonDict]
+Json = Union[JsonDict, JsonList]
```

Comment:
Well a string and a number and null(None) can also be JSON. Could a more adequate name be used?
Sorry I find these annotations a bit confusing.

Reply:
I wanted to have aliases to denote what raw data as the one returned e.g. by ``json.load(...)``. Then I wanted to distinguish when the data is like ``{ "x": 2, ... }`` (classic json) from ``[ {...}, {...}]`` (list of jsons). It is true that is not json per-se but I did not find a better name so far.  How would you call it? :-)  Any suggestions?

Reply:
why not AnyDict and AnyDictList?